src/components/sbb-form-field/readme.md

```diff
@@ -71,8 +71,14 @@ element. However, several of the form field's optional features interact with th
 contained within the form field.
 
 When you provide a label via `<label>` or the `label` attribute, `<sbb-form-field>` automatically
-associates this label with the field's form element via a native <label> element, using the `for`
+associates this label with the field's form element via a native `<label>` element, using the `for`
 attribute to reference the control's ID.
+When using a non-native form element (e.g. `<sbb-select>`) `aria-label` is used, to connect the
```

Comment:
```suggestion
When using a non-native form element (e.g. `<sbb-select>`) `aria-labelledby` is used, to connect the
```

Reply:
Nope, that's a mistake. Brain might have been afk. 😄